usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -568,6 +568,9 @@ def load_file_c(submission_attributes, db_cursor, certified_award_financial):
         award = find_matching_award(**filters)
 
         if award:
+            # Mark the award as updated, so it will be reloaded into ElasticSearch during nightly job
+            award.updated_at = datetime.now(timezone.utc)
+            award.save()
```

Comment:
What are your thoughts on performing a bulk update instead of updating them one at a time?  Maybe something like `Award.objects.filter(id__in=awards_touched).update(updated_at=datetime.now(timezone.utc))` (untested) outside of the loop?  Or maybe `Award.objects.filter(id__in=set(a.id for a in awards_touched)).update(updated_at=datetime.now(timezone.utc))` (untested) if we think there are going to be repeated award ids.

Reply:
I think this is a good call Kirk! I need to step up my Django game!